Artifacts/PgSql/Data/Security/1060-RemoveRedundantBootstrapPermissions.sql

```diff
@@ -14,35 +14,28 @@
 
     BEGIN
 
-        SELECT AuthorizationStrategyId INTO authorizationStrategy_Id FROM dbo.AuthorizationStrategies WHERE DisplayName='No Further Authorization Required';
-        SELECT ActionId INTO createAction_Id FROM dbo.Actions WHERE ActionName='Create';
-
-        IF NOT EXISTS (SELECT 1 FROM dbo.ResourceClaims WHERE ClaimName = claim_Name)
-        THEN
-            RAISE EXCEPTION 'ClaimName ''%'' not found.', claim_Name;
-        END IF;
-
-        IF NOT EXISTS (SELECT 1 FROM dbo.ClaimSets WHERE ClaimSetName = claimSet_Name)
-        THEN
-            RAISE EXCEPTION 'ClaimSetName ''%'' not found.', claimSet_Name;
-        END IF;
-
-        SELECT ResourceClaimId INTO educationOrganizationsResourceClaimId
-        FROM dbo.ResourceClaims
-        WHERE ClaimName = claim_Name;
-
-        SELECT claimSetId INTO claimSet_Id
-        FROM dbo.ClaimSets
-        WHERE ClaimSetName = claimSet_Name;
-
-        DELETE FROM dbo.ClaimSetResourceClaims 
-        WHERE ClaimSetResourceClaimId IN  
-            (SELECT ClaimSetResourceClaimId  FROM dbo.ClaimSetResourceClaims 
-            WHERE ClaimSet_ClaimSetId = claimSet_Id
-                AND Action_ActionId = createAction_Id
-                AND AuthorizationStrategyOverride_AuthorizationStrategyId = authorizationStrategy_Id
-                AND ResourceClaim_ResourceClaimId IN 
-                    (SELECT  ResourceClaimId  FROM dbo.ResourceClaims 
-                    WHERE ParentResourceClaimId = educationOrganizationsResourceClaimId));
+    SELECT AuthorizationStrategyId INTO authorizationStrategy_Id FROM dbo.AuthorizationStrategies WHERE DisplayName='No Further Authorization Required';
```

Comment:
We lost a lot of indent here with this change, are we keeping everything indented under begin, and then again under the if check?

Reply:
Now Inverted what was before and aligned with new changes